pkg/disttask/framework/taskexecutor/manager.go

```diff
@@ -91,6 +92,22 @@ func NewManager(ctx context.Context, id string, taskTable TaskTable) (*Manager,
 	if totalCPU <= 0 || totalMem <= 0 {
 		return nil, errors.Errorf("invalid cpu or memory, cpu: %d, memory: %d", totalCPU, totalMem)
 	}
+	cgroupLimit, version, err := cgroup.GetCgroupMemLimit()
+	// ignore the error of cgroup.GetCgroupMemLimit, as it's not a must-success step.
+	if err == nil && version == cgroup.V2 {
```

Comment:
From the code, it seems only the Cgroup V2 return mem limit should be adjusted.

Reply:
yes, v1 returns the hierarchy mem limit which is correct